src/languages/maxima.js

```diff
@@ -0,0 +1,415 @@
+      + ' simpsum sinnpiflag solvedecomposes solveexplicit solvefactors solvenullwarn'
+      + ' solveradcan solvetrigwarn space sparse sphere spring_embedding_depth sqrtdispflag'
+      + ' stardisp startphi starttheta stats_numer stringdisp structures style sublis_apply_lambda'
+      + ' subnumsimp sumexpand sumsplitfact surface surface_hide svg_file symmetric'
+      + ' tab taylordepth taylor_logexpand taylor_order_coefficients taylor_truncate_polynomials'
+      + ' tensorkill terminal testsuite_files thetaresolution timer_devalue title tlimswitch'
+      + ' tr track transcompile transform transform_xy translate_fast_arrays transparent'
+      + ' transrun tr_array_as_ref tr_bound_function_applyp tr_file_tty_messagesp tr_float_can_branch_complex'
+      + ' tr_function_call_default trigexpandplus trigexpandtimes triginverses trigsign'
+      + ' trivial_solutions tr_numer tr_optimize_max_loop tr_semicompile tr_state_vars'
+      + ' tr_warn_bad_function_calls tr_warn_fexpr tr_warn_meval tr_warn_mode'
+      + ' tr_warn_undeclared tr_warn_undefined_variable tstep ttyoff tube_extremes'
+      + ' ufg ug %unitexpand unit_vectors uric uriem use_fast_arrays user_preamble'
+      + ' usersetunits values vect_cross verbose vertex_color vertex_coloring vertex_partition'
+      + ' vertex_size vertex_type view warnings weyl width windowname windowtitle wired_surface'
+      + ' wireframe xaxis xaxis_color xaxis_secondary xaxis_type xaxis_width xlabel'
+      + ' xlabel_secondary xlength xrange xrange_secondary xtics xtics_axis xtics_rotate'
+      + ' xtics_rotate_secondary xtics_secondary xtics_secondary_axis xu_grid x_voxel'
+      + ' xy_file xyplane xy_scale yaxis yaxis_color yaxis_secondary yaxis_type yaxis_width'
+      + ' ylabel ylabel_secondary ylength yrange yrange_secondary ytics ytics_axis'
+      + ' ytics_rotate ytics_rotate_secondary ytics_secondary ytics_secondary_axis'
+      + ' yv_grid y_voxel yx_ratio zaxis zaxis_color zaxis_type zaxis_width zeroa zerob'
+      + ' zerobern zeta%pi zlabel zlabel_rotate zlength zmin zn_primroot_limit zn_primroot_pretest';
+
+  return {
+    lexemes: '[A-Za-z_%][0-9A-Za-z_%]*',
+    keywords: {
+      keyword: KEYWORDS,
+      literal: LITERALS,
+      built_in: BUILTIN_FUNCTIONS,
+      symbol: BUILTIN_VARIABLES
+    },
+    contains: [
+      {
+        // C-style comments, with nonstandard color.
+        // Dunno if this recognizes nested comments.
+        // I wonder, should we try to recognize quoted strings in comments?
+        className: 'meta',
+        begin: '/\\*',
+        end: '\\*/'
+      },
+      hljs.QUOTE_STRING_MODE,
+      {
+        className: 'number',
+        variants: [
+          {
+            // float number w/ exponent
+            // hmm, I wonder if we ought to include other exponent markers?
+            begin: '\\b(\\d+|\\d+\\.|\\.\\d+|\\d+\\.\\d+)[Ee][-+]?\\d+\\b',
+            relevance: 1
```

Comment:
We usually try to avoid relevances different from 0 and 10 because so precise tuning causes problems with next merged languages. I'll try to play with this.